scripts/spec/kickstart_generator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+require 'kickstart_generator'
+
+describe Build::KickstartGenerator do
+  describe "#run" do
+    def build_base
+      data_file_path("kickstart_generator")
+    end
+
+    def generated
+      "#{build_base}/kickstarts/generated"
+    end
+
+    def ks_text
+      File.read("#{generated}/base.ks")
+    end
+
+    after(:each) do
+      FileUtils.rm_f(Dir.glob("#{generated}/*"))
```

Comment:
see above note. (putting dirname into a let makes for easy access at cleanup time)